content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system/content.md

```diff
@@ -0,0 +1,495 @@
+
+The project shown is intended to replicate a scale smart farming application, it can be implemented on real agriculture fields using the same hardware and firmware.
+
+## Goals
+
+The goal of this application note is to showcase a smart farming irrigation system using a combination of an Edge Control, an MKR WiFi 1010, and the Arduino IoT Cloud. The project's objectives are the following:
+
+- Independently control 4 irrigation zones using motorized ball valves.
+- Get water from a smart tank with water level monitoring.
+- Program irrigation timers from remote through Arduino IoT Cloud by using WiFi connectivity.
+- Manually activate irrigation from Arduino IoT Cloud through dedicated widgets.
+- Monitor average irrigation time and water consumption on dedicated charts on Arduino IoT Cloud.
+- Provide a weather station on Arduino IoT Cloud, using API connected to a weather website to decide on whether to irrigate or not.
+
+## Hardware and Software Requirements
+
+![Project main hardware and materials](assets/hardware_16-9.png)
+
+### Hardware Requirements
+- Arduino Edge Control
+- Arduino MKR WiFi 1010
+- Arduino Edge Control Enclosure Kit
+- 4-20mA (0-1 meter) Liquid Level Sensor
+- 4x 2 Wires Motorized ball Valves (3 Wires version are also supported)
+- 12V DC power supply 
+- 11' of 1/2" PVC pipes
+- 3x 1/2" PVC TEE
+- 8x 1/2" PVC elbow
+- 1/2" Manual Valve
+- 4x 1/2" PVC caps
+- 11x 1/2" PVC male adapters
+- 7x 1/2" wall pipe brackets
+- 4x Rectangular planters
+- DIN rail
+- 6x Cable glands
+- 20' Duplex cable AWG 18
+- Electrical Register Box
+- Water Tank
+
+### Software Requirements
+
+- [Arduino IDE 1.8.10+](https://www.arduino.cc/en/software), [Arduino IDE 2](https://www.arduino.cc/en/software), or [Arduino Web Editor](https://create.arduino.cc/editor).
+- If you are going to use an offline Arduino IDE, you must install the following libraries: `Arduino_EdgeControl`, `ArduinoIoTCloud`, `Arduino_JSON`, `ArduinoJson`, `ArduinoHttpClient` and `Arduino_ConnectionsHandler`. You can install them using the Arduino IDE Library Manager.
+- The [smart irrigation system codes](assets/Edge-Control_MKR_Codes.zip).
+- [Arduino Create Agent](https://create.arduino.cc/getting-started/plugin/welcome) to add the Arduino Portenta H7 to the Arduino IoT Cloud.
+
+## Smart Irrigation System Setup
+
+The electrical connections of the intended application are shown in the diagram below:
+
+![Electrical connections of the irrigation system](assets/wiring-diagram-1.png)
+
+The Edge Control board will be powered with an external 12V DC power supply connected to BATT+ and GND of J11 respectively.
+The four motorized ball valves will be connected to the Edge Control Latching outputs of J9 connector from OUT0 to OUT6.
+The water level transmitter will be connected to the +19V reference and the 4-20mA input number 1 of the J7 connector.
```

Comment:
The connection diagram with a brief explanation looks good to me. It is using 2-wire motorized valve, but I think it could be nice to include this [Connecting and Controlling a Motorized Ball Valve tutorial](https://docs.arduino.cc/tutorials/edge-control/motorized-ball-valve) as a reference (3-wire use case) for the users. Also, as an additional suggestion, I think it could be nice to "magnify" the connection diagram, especially for 2-wire valve, to explicitly show the connection setup.

Reply:
I added the reference to the 3 wires guide. Just left pending to add the connection close up picture.